build.gradle

```diff
@@ -19,14 +19,13 @@ repositories {
 }
         
 dependencies {
-    compileOnly group: 'se.curity.identityserver', name: 'identityserver.sdk', version:'4.2.1'
-    compileOnly group: 'org.slf4j', name: 'slf4j-api', version:'1.7.22'
-    compileOnly group: 'org.jetbrains.kotlin', name: 'kotlin-stdlib', version:'1.3.41'
-    compileOnly group: 'org.hibernate', name: 'hibernate-validator', version:'5.1.3.Final'
+    compileOnly group: 'se.curity.identityserver', name: 'identityserver.sdk', version:'8.0.0'
+    compileOnly group: 'org.slf4j', name: 'slf4j-api', version:'2.0.3'
+    compileOnly group: 'org.jetbrains.kotlin', name: 'kotlin-stdlib-jdk8', version:'1.6.10'
```

Comment:
why use `kotlin-stdlib-jdk8` instead of `kotlin-stdlib`?

Reply:
That is the one provided by the Curity Identity Server: https://curity.io/docs/idsvr/latest/developer-guide/plugins/index.html#kotlin-standard-library We could depend on stdlib in version >= 1.8 but then the dependencies would have to be added separately with the plugin. I think we can update that once kotlin version is updated in the main product.